autopilot/contractor.go

```diff
@@ -562,6 +582,71 @@ func (c *contractor) initialContractFunding(settings rhpv2.HostSettings, txnFee,
 	return funding
 }
 
+func (c *contractor) renterFundsEstimate(cfg Config, contract worker.Contract, isRefresh bool) (estimate types.Currency, err error) {
+	if isRefresh {
+		estimate, err = c.refreshFundingEstimate(cfg, contract)
+		if err != nil {
+			c.logger.Errorw(
+				fmt.Sprintf("could not get refresh funding estimate, err: %v", err),
+				"hk", contract.HostKey(),
+				"fcid", contract.ID,
+			)
+		}
+	} else {
+		estimate, err = c.renewFundingEstimate(cfg, contract)
+		if err != nil {
+			c.logger.Errorw(
+				fmt.Sprintf("could not get renew funding estimate, err: %v", err),
+				"hk", contract.HostKey(),
+				"fcid", contract.ID,
+			)
+		}
+	}
+	return
```

Comment:
Maybe like this?

```go
var estimate types.Currency
var err error
if isRefresh {
  estimate, err = c.refreshFundingEstimate(cfg, contract)
}else {
  estimate, err = c.renewFundingEstimate(cfg, contract)
}
if err != nil {
  fmt.Sprintf("could not get renew funding estimate, err: %v", err),
 				"hk", contract.HostKey(),
 				"fcid", contract.ID,
                                "refresh", isRefresh,
}
```

Reply:
did it slightly different but implemented your remark